modules/mod-pcm/tests/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+add_unit_test(
+   NAME
+      mod-pcm
+   SOURCES
+      GetAcidizerTagsTests.cpp
+   LIBRARIES
+      mod-pcm
+      lib-file-formats
+)
```

Comment:
Why does a module have a unit test?  Can this go to `lib-file-formats`?

I like a banner comment in each `CMakeLists.txt` delimited by
```
#[[
   ...
]]
```

Reply:
See other examples.  That banner just summarizes the purpose of the library.

Reply:
I'm failing to configure the build for macOS:
```
 Target "mod-pcm" of type MODULE_LIBRARY may not be linked into another
  target.  One may link only to INTERFACE, OBJECT, STATIC or SHARED
  libraries, or to executables with the ENABLE_EXPORTS property set.
```

Reply:
Yes: see fe2caec